studies/templates/emails/notify_admins_of_study_action.txt

```diff
@@ -11,9 +11,9 @@ Dear {{ org_name }} Admin,
 {% elif action == 'paused' %}
     {{ researcher_name }} has paused the study {{study_name}}. {{base_url}}{% url 'exp:study-detail' pk=study_id %}
 {% elif action == 'deactivated' %}
-    {{ researcher_name }} has deactivated the study {{study_name}}. {{base_url}}{{% url 'exp:study-detail' pk=study_id %}
+    {{ researcher_name }} has deactivated the study {{study_name}}. {{base_url}}{% url 'exp:study-detail' pk=study_id %}
 {% elif action == 'deployed' %}
-    {{ study_name }} has been deployed. {{base_url}}{{% url 'exp:study-detail' pk=study_id %}
+    {{ study_name }} has been deployed. {{base_url}}{% url 'exp:study-detail' pk=study_id %}
```

Comment:
Thanks for fixing these by the way. It was hard to see the extra curly in the console backend printout.